tensorzero-core/src/variant/mixture_of_n.rs

```diff
@@ -80,13 +80,23 @@ pub struct UninitializedFuserConfig {
 }
 
 impl UninitializedMixtureOfNConfig {
-    pub fn load(self, schemas: &SchemaData) -> Result<MixtureOfNConfig, Error> {
+    pub fn load(
+        self,
+        schemas: &SchemaData,
+        error_context: &ErrorContext,
+    ) -> Result<MixtureOfNConfig, Error> {
         Ok(MixtureOfNConfig {
             weight: self.weight,
             timeout_s: self.timeout_s,
             candidates: self.candidates,
             fuser: FuserConfig {
-                inner: self.fuser.inner.load(schemas)?,
+                inner: self.fuser.inner.load(
+                    schemas,
+                    &ErrorContext {
+                        function_name: error_context.function_name.clone(),
+                        variant_name: format!("{}.fuser", error_context.variant_name),
```

Comment:
this feels like slightly a hack but I don't think there's a simple + elegant way to do this here